### DISCUSSION TITLE
app is crashing on specific gif file

### DISCUSSION THREAD

**Discussion by fenio** [community/community#77803] [Mobile]
I don't really know what would be the best place to submit that issue so I'm doing it here.

If you have Github app installed on the most recent IOS and you go to the following project https://github.com/AmrDeveloper/GQL and click on demo then it crashes. If you remove the GH app and do the same then demo shows up in Safari but if you try to copy&paste that GIF into let's say Telegram then it shows just circle with progress but it's not able to actually paste it embedded. 
There's definitely something weird happening around that GIF. 

Just try to play around https://github.com/AmrDeveloper/GQL/blob/master/media/gql_demo.gif

**Comment by fenio** [comment]
And to be honest I don't really think this is a problem with Github itself but rather with IOS. But since I discovered it using GH I decided to submit it here first.

**Comment by fenio** [comment]
This has been also submitted to Apple as part of FB13440167.

**Comment by dakshithadissanayaka** [comment]
It appears there might be an issue with the GIF file itself or its rendering within the GitHub app and Safari. The fact that the GIF displays correctly in Safari after removing the GitHub app suggests a compatibility issue with the app's handling of the GIF format.

To effectively address this issue, it's recommended to report it to both GitHub and the developer of the GQL project. Here's how to proceed:

**Reporting to GitHub:**

1. **GitHub Support:** Contact GitHub support directly through their support channels, such as the GitHub Help Center ([https://support.github.com/](https://support.github.com/)) or their support email address ([https://support.github.com/](https://support.github.com/)). Provide a detailed description of the issue, including steps to reproduce it, the iOS version you're using, and the specific GIF URL.

2. **GitHub Issue Tracker:** Create a new issue on the GitHub repository for the GitHub app ([https://github.com/android/.github/issues](https://github.com/android/.github/issues)). In the issue description, clearly explain the problem, provide the GIF URL, and mention the iOS version you're using.

**Reporting to GQL Project Developer:**

1. **GitHub Issues Tracker:** Create a new issue on the GitHub repository for the GQL project ([https://github.com/BoD/android-graphql-sample](https://github.com/BoD/android-graphql-sample)). In the issue description, clearly explain the problem, provide the GIF URL, and mention the iOS version you're using.

2. **Project Communication Channels:** If the GQL project has other communication channels, such as a forum or Discord server, you can also report the issue there. This allows you to reach out to the project maintainers directly and potentially get a faster response.

By reporting the issue to both GitHub and the GQL project developer, you increase the chances of getting it addressed promptly and effectively.

**Comment by samus-aran** [comment]
Hi @fenio ! We hope you found what you were looking for. ✨ Others can search for this discussion if they have similar issues.

We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
